specification/containerservice/resource-manager/Microsoft.ContainerService/stable/2021-03-01/examples/ContainerServiceGetOSOptions.json

```diff
@@ -0,0 +1,28 @@
+{
+  "parameters": {
+    "api-version": "2021-03-01",
+    "subscriptionId": "subid1",
+    "location": "location1"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "/subscriptions/subid1/providers/Microsoft.ContainerService/locations/location1/os-options",
```

Comment:
The "id" must have the name as the last segment ("default" in this case)

Reply:
This is basically a fixed metadata-type of return data. It is not diff by name. So here, the name is not very relevant.

Reply:
Relevant or not to you it is VERY relevant to the platform as a whole.  IDs follow a very specific convention that must be adhered to.  https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/resource-api-reference.md#get-resource

Reply:
I am not seeing any update here.

---
In reply to: [603612699](https://github.com/Azure/azure-rest-api-specs/pull/13619#discussion_r603612699) [](ancestors = 603612699)

Reply:
Updated to add /default in the name path.